charts/uptime-probe/templates/alertrules.yaml

```
@@ -18,6 +18,7 @@ spec:
       for: 5m
       labels:
         severity: warning
+        origin: w3f
```

The chart is now deployed to w3f but could be deployed elsewhere, we need to use a value here. Also add a default to values.yaml.
```suggestion
        origin: {{ .Values.origin }}
```